resources/lang/de/Widget.properties

```
@@ -703,6 +705,14 @@ itemDataTableItemAgeRating = "Altersfreigabe"
 itemDataTableItemExternalId = "Externe Varianten-ID"
 itemDataTableItemVariationModel = "Modell"
 itemDataTableItemManufacturer = "Hersteller"
+itemManufacturerLabel = "Item manufacturer"
```

Here, you added the English translations to the DE properties file (and vice versa)

```suggestion
itemManufacturerLabel = "Artikelhersteller"
itemManufacturerSampleName = "Name 1"
itemManufacturerSampleCompanyName = "Musterhersteller"
itemManufacturerSampleAddress = "Musterstraße 12"
itemManufacturerSampleLocation = "12209 Musterstadt Deutschland"
itemManufacturerSampleDomain = "info@musterdomain.com"
itemManufacturerSampleHomepage = "www.musterhomepage.com"
itemManufacturerSamplePhone = "+123456789"
```